cmd/kube-controller-manager/app/discovery.go

```diff
@@ -24,11 +24,13 @@ import (
 	"net/http"
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/klog"
 	endpointslicecontroller "k8s.io/kubernetes/pkg/controller/endpointslice"
 )
 
 func startEndpointSliceController(ctx ControllerContext) (http.Handler, bool, error) {
-	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "discovery", Version: "v1alpha1", Resource: "endpointslices"}] {
+	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "discovery.k8s.io", Version: "v1alpha1", Resource: "endpointslices"}] {
```

Comment:
please use `discoveryv1alpha1.Resource("endpointslices")` to avoid these typos

Reply:
Good catch, thanks! Since this requires a `GroupVersionResource` instead of just a `GroupResource`, the best alternative I could come up with here was `discoveryv1alpha1.SchemeGroupVersion.WithResource("endpointslices")`.